assets/components/contributionSelection/contributionSelectionReducer.js

```diff
@@ -154,7 +152,11 @@ function contributionSelectionReducerFor(scope: string, countryGroupId: CountryG
         };
 
       case 'SET_AMOUNT':
-        return updatePredefinedAmount(state, action.amount);
+        return updatePredefinedAmount(state, state.contributionType, action.amount);
```

Comment:
any reason why we're using `state.contributionType ` here and `action.contributionType` for the next case?